browser-test/src/support/applicant_questions.ts

```diff
@@ -36,17 +38,18 @@ export class ApplicantQuestions {
     await this.page.waitForSelector('.cf-address-street-1 input')
     await this.page.waitForSelector('.cf-address-street-2 input')
     await this.page.waitForSelector('.cf-address-city input')
-    await this.page.waitForSelector('.cf-address-state input')
     await this.page.waitForSelector('.cf-address-zip input')
 
     // Check values are equal to expected
     await this.validateInputValue(street, '.cf-address-street-1 input')
     await this.validateInputValue(line2, '.cf-address-street-2 input')
     await this.validateInputValue(city, '.cf-address-city input')
-    await this.validateInputValue(state, '.cf-address-state input')
+    await this.validateDropdownValue(state, '.cf-address-state')
     await this.validateInputValue(zip, '.cf-address-zip input')
   }
-
+  async validateDropdownValue(value: string, dropdownId: string) {
```

Comment:
Nit: Can you move this down to be with the other methods that validate input so they stay together?

Reply:
sure